charts/secretsmanagement-operator/templates/preinst-autodetect-audience-job.yaml

```diff
@@ -24,11 +24,15 @@ spec:
         app.kubernetes.io/instance: {{ .Release.Name | quote }}
         helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     spec:
+      {{- if .Values.global.imagePullSecrets }}
+      imagePullSecrets:
+      {{- include "image-pull-secrets" . |nindent 6 }}
+      {{- end }}
       restartPolicy: Never
       containers:
       - name: autodetect-audience
         # TODO[FH]: remove prereleasesuffix
```

Comment:
Oops my fault...
Can you please remove this TODO[FH] comment, the prerelease suffix was removed.   :-)